entrypoint.sh

```diff
@@ -183,24 +266,80 @@ fi
 touch "${WDA_LOG_FILE}"
 # verify if wda is already started and reuse this session
 
+runWda() {
+  declare -i index=0
+  isWdaStarted=0
+  while [[ $index -lt 10 ]]; do
+    if ! (pgrep -f "ios runwda" > /dev/null 2>&1); then
+      ios runwda \
+        --env USE_PORT="$WDA_PORT" \
+        --env MJPEG_SERVER_PORT="$MJPEG_PORT" \
+        --env UITEST_DISABLE_ANIMATIONS=YES \
+        --log-output="${WDA_LOG_FILE}" \
+        --udid="$DEVICE_UDID"
+      logger "WARN" "'ios runwda' process broke. Attempt to recover."
+      isWdaStarted=0
+    else
+      logger "WARN" "WDA already started"
+      isWdaStarted=1
+      break
+    fi
+    sleep 1
+    index+=1
+  done; index=0
+
+  if [[ $isWdaStarted -eq 0 ]]; then
+    logger "ERROR" "Can't run WDA. Restarting!"
+    exit 1
```

Comment:
any sense to return error code if 10 times we couldn't start wda? how about exit 0?